docs/documentation/server_development/topics/themes.adoc

```diff
@@ -241,6 +241,19 @@ Or to use directly in HTML templates add the following to a custom HTML template
 <img src="${url.resourcesPath}/img/image.jpg">
 ----
 
+==== Adding an image to a email theme
+
+To make images available to the theme add them to the `<THEME TYPE>/email/resources/img` directory of your theme. These can be used from within directly in HTML templates.
+
+For example to add an image to the `mytheme` copy an image to `themes/mytheme/email/resources/img/logo.jpg`.
+
+To use directly in HTML templates add the following to a custom HTML template:
+
+[source,html]
+----
+<img src="${url.resourcesUrl}/img/image.jpg">
```

Comment:
Images should always be provided with an `alt` text, this aids user with a visual impairment. The exception to this are [decorative images](https://www.w3.org/WAI/tutorials/images/decorative/).

Reply:
Hello @jonkoops 😄 , thanks for your feedback.  I've just completed the documentation because there's a difference in the variable to be used in the case of an email template.
I agree with you about the alt parameter, but it doesn't appear in the example given for the custom template either. I've just used the same example to show that the variable is different. Or should I add the alt for the other example too? But what to put? This parameter is specific to HTML code and has nothing to do with Keycloak from my point of view.

Reply:
> I've just completed the documentation because there's a difference in the variable to be used in the case of an email template (url.resourcesPath vs url.resourcesUrl).

Hmmm, this is then likely a bug, we should try to unify the names of these variables as much as possible, that would remove the need to document them separately.

> Or should I add the alt for the other example too?

Yeah, ideally all images that are not decorative should have a descriptive text. But that could be a seperate PR if you like.

> This parameter is specific to HTML code and has nothing to do with Keycloak from my point of view.

Well, it does, these templates are rendered in HTML. We have a duty to make sure our code is accessible, and these best practices should also be communicated in our documentation. Also, some e-mail clients will block images by default, so they would look quite broken without an alt-text.

Reply:
> Hmmm, this is then likely a bug, we should try to unify the names of these variables as much as possible, that would remove the need to document them separately.

For me, there's a real contextual difference that justifies a different name. When rendering on the url of a Keycloak server, you can use the path (url.resourcesPath) but when you're in the context of an email . Relative paths may not work reliably in email clients. This is because the email display is not an HTML page rendered on the Keycloak server. I don't know if I'm making myself clear. 😄 

> Well, it does, these templates are rendered in HTML. We have a duty to make sure our code is accessible, and these best practices should also be communicated in our documentation. Also, some e-mail clients will block images by default, so they would look quite broken without an alt-text.

Very well, I agree with you, I can correct my example. Should I also correct the previous example in the same PR?

Reply:
Yes, I agree that makes sense. Feel free to update the other example as well.

Reply:
It's done 😄